### PR TITLE
Add Tweaks option to use old photo layout margins

### PR DESCRIPTION
**Pull request by dlmarquis**

### CODE REVIEW

File: Extensions/tweaks.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
-
 //* TITLE Tweaks **//
-//* VERSION 5.0.1 **//
+//* VERSION 5.0.2 **//
```
Comment (Wolvan) [inline Extensions/tweaks.js]: this is a ``minor`` version bump, not a ``patch``. You are adding new functionality

**Comment by Wolvan** [comment]
You don't have a post listener, so the function only gets called on the start of tweaks. That leads to the design not being applied on subsequently loaded posts

**Comment by dlmarquis** [comment]
Oh fuck right.

File: Extensions/tweaks.js [hunk 2]
```diff
@@ -626,6 +634,31 @@ XKit.extensions.tweaks = new Object({
...
+		$(".post_media_photo.image").each(function() {
+			$(this).attr("style","margin-left: 20px; width: 500px;");
+		});
+
+		$(".photoset_row").each(function() {
+			var photoset_row = $(this);
+			photoset_row.attr("style","margin-left: 20px; margin-bottom: 10px;");
+			photoset_row.find("img").each(function() {
+				var img = $(this);
+				var imgstyle = img.attr("style").trim().slice(-12);
```
Comment (nightpool) [inline Extensions/tweaks.js]: this is fragile. can't you use `jQuery.css('width')` instead? Or just .width()?

**Comment by nightpool** [comment]
are those really the only three widths of images tumblr supports? Also shouldn't that be margin: auto rather then margin: right?

**Comment by dlmarquis** [comment]
@nightpool No, margin:auto doesn't work for me.

**Comment by nightpool** [comment]
@dlmarquis ah you're just applying a margin of half of the removed width. that makes sense.

**Comment by dlmarquis** [comment]
Goddammit. Android git does not appreciate rebasing.

**Comment by hobinjk** [comment]
Works locally and LGTM.
@homu r+

**Pull request merged**